src/test/unit/translations.spec.ts

```diff
@@ -65,4 +65,27 @@ describe('Check missing keys in translation files', () => {
 
     expect(findMissingKeys(englishContents, welshContents)).toEqual([]);
   });
+
+  function checkWelshTranlationFile(obj: { [index: string]: unknown }, full_key: string) {
+    for (const k in obj) {
+      if (typeof obj[k] === 'string') {
+        const val: string = obj[k] as string;
+        if (val.includes('Welsh Translation required') || val === '') {
```

Comment:
There is another scenario I've discovered which is the text includes '- Requires Update'

Reply:
'Update Required' also presents in jsons

Reply:
done